WordPress/src/main/java/org/wordpress/android/ui/bloggingprompts/onboarding/BloggingPromptsOnboardingUiStateMapper.kt

```diff
@@ -27,7 +27,9 @@ class BloggingPromptsOnboardingUiStateMapper @Inject constructor() {
         )
 
         val trailingLabel = UiStringPluralRes(
-                R.plurals.my_site_blogging_prompt_card_number_of_answers,
+                0,
```

Comment:
What does this mean to use a resource of `0` for the `zeroRes` parameter, does thus automatically use the value for the `oneRes` as a fallback?

From what I read [in `ResourceProvider.getQuantityString` here](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/WordPress/src/main/java/org/wordpress/android/viewmodel/ResourceProvider.kt#L45-L50), there doesn't seem to be a special case for that, and I'm assuming `getString(0)` will thus return `nil` instead of returning a copy for "no answers"… which I guess means that there won't be any text at all?

(While by contrast, if GlotPress support allowed us to be able to use the native support for `<plurals>` resources here, the Android runtime would pick the right copy as a fallback if none was defined for the `zero` case, I think?)

Reply:
Oh.
> place the smartphone date in the past to find a prompt without answers, notice that no avatar nor "# answer" is presented

I guess having no text for the case of zero answers _is_ the intended design after all — and thus why it's ok to use `0` there (not as a mean to say "we don't have a specific copy for zero so let Android runtime use its better judgment to pick the right plural alternative", but as a mean to say "for the case zero, return a `nil` string and don't show any text) 👍

Reply:
>I guess having no text for the case of zero answers is the intended design [...]

yep 👆...and thanks for always dig into details @AliSoftware 🙇